src/blenderbim/blenderbim/bim/import_ifc.py

```diff
@@ -684,6 +686,62 @@ def create_structural_point_connections(self):
             obj.matrix_world = self.apply_blender_offset_to_matrix_world(obj, placement_matrix)
             self.link_element(product, obj)
 
+    def create_pointclouds(self, products):
+        result = set()
+        for product in products:
+            if not self.is_pointcloud(product):
+                continue
+            items = []
+            for representation in product.Representation.Representations:
+                if representation.RepresentationType == 'PointCloud':
+                    items.extend(representation.Items)
+                elif self.file.schema == "IFC2X3" and representation.RepresentationType == 'GeometricSet':
+                    items.extend(representation.Items)
+
+            placement_matrix = ifcopenshell.util.placement.get_local_placement(product.ObjectPlacement)
```

Comment:
In theory it's more complex than this, for example if a mapped representation is used, it may apply another coordinate transformation to the coordinates themselves. However, I'm happy to turn a blind eye to this because I find it oh-so-messy-and-confusing.